contrib/mariner/aziot-identity-service.spec

```diff
@@ -57,6 +58,15 @@ This package contains development files for the Azure IoT device runtime.
 
 
 %prep
+# include rust toolchain that matches the one from iot-identity-service's pipeline
+pushd ~
+tar xf %{SOURCE3} --no-same-owner --strip-components=1
+popd
+export CARGO_HOME=~/.cargo
+export PATH=$PATH:$CARGO_HOME/bin
+export RUSTUP_HOME=~/.rustup
+export RUSTUP_TOOLCHAIN=~/.rustup/toolchains/1.47-x86_64-unknown-linux-gnu
```

Comment:
Why do you need this? When `cargo` is invoked in the source directory (as part of `make ...`), it will already find the `rust-toolchain` file and know which toolchain to use.

Reply:
without explicitly setting the toolchain i get the error bellow when using cargo

time="2022-03-23T12:32:34-04:00" level=debug msg="+ cargo install bindgen --path rust-bindgen-0.54.0 --offline"
time="2022-03-23T12:32:34-04:00" level=debug msg="error: no override and no default toolchain set"

Reply:
Like I said, make sure to `cd` into the source directory, ie the root of the repository, ie the directory that contains the `rust-toolchain` file, first.

Reply:
thanks for the info. this should be fixed in the spec file now.